react-basic-practice/src/types/error.ts

```diff
@@ -0,0 +1,3 @@
+export type Errors = {
```

Comment:
naming is not good. Pls update naming and verify all source code and use this type for similar places

Reply:
i done